open_event/static/js/admin/event/scheduling.js

```diff
@@ -0,0 +1,502 @@
+            $(".no-sessions-info").hide();
+        }
+        $sessionElement.ellipsis().ellipsis();
+    });
+    fixOverlaps();
+    $("[data-toggle=tooltip]").tooltip();
+}
+
+function loadData(eventId, callback) {
+    $.get("https://open-event.herokuapp.com/api/v2/events/" + eventId + "/tracks", function (tracks) {
+        $.get("https://open-event.herokuapp.com/api/v2/events/" + eventId + "/sessions", function (sessions) {
+            processTrackSession(tracks, sessions, callback);
+        });
+    });
+}
+
+$(document).on("click", ".date-change-btn", function () {
+    $(this).addClass("active");
+    loadTracksToTimeline($(this).text());
+    $(this).siblings().removeClass("active");
+});
+
+$(document).on("click", ".session.scheduled > .remove-btn", function () {
+    var $sessionElement = $(this).parent();
+    $sessionElement.appendTo($(".sessions-holder"));
+    $sessionElement.addClass("unscheduled").removeClass("scheduled").tooltip("hide").attr("data-original-title", "");
+    $sessionElement.css({
+        "-webkit-transform": "",
+        "transform": "",
+        "background-color": ""
+    }).removeData("x").removeData("y");
+});
+
+function generateTimeUnits() {
+    var start = moment.utc().hour(time.start.hours).minute(time.start.minutes).second(0);
+    var end = moment.utc().hour(time.end.hours).minute(time.end.minutes).second(0);
+    var $timeUnitsHolder = $(".timeunits");
+    var timeUnitsCount = 1;
+    while (start <= end) {
+        var timeUnitDiv = $("<div class='timeunit'>" + start.format('HH:mm') + "</div>");
+        $timeUnitsHolder.append(timeUnitDiv);
+        start.add(time.unit.minutes, 'minutes');
+        timeUnitsCount++;
+    }
+    $(".track-container").css("height", timeUnitsCount * time.unit.pixels);
+
+    time.unit.count = timeUnitsCount;
+}
+
+$(document).ready(function () {
+
+    generateTimeUnits();
+
+    // TODO Load the event ID dynamically based on current event
+    loadData(18, function () {
+        $(".flash-message-holder").hide();
+        $(".scheduler-holder").show();
+        initializeInteractables();
+
+    });
+
+    $(".clear-overlaps-button").click(function () {
+        fixOverlaps();
+    });
+
+    $(".timeline").scroll(function () {
+        var cont = $(this);
+        var el = $(cont.find(".track-inner")[0]);
+        var elementTop = el.position().top;
+        var pos = cont.scrollTop() + elementTop;
+        cont.find(".track-header").css("top", pos + "px");
+    });
+
+    var sessionTemplate = $("#session-template").html();
+    var $unscheduledSessionsHolder = $(".sessions-list");
+
+    $("#sessions-search").valueChange(function (value) {
+
+        var filtered = [];
+        if (_.isEmpty(value) || value == "") {
+            filtered = currentUnscheduledStore;
+        } else {
+            filtered = _.filter(currentUnscheduledStore, function (session) {
+                return fuzzy_match(session.title, value);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'fuzzy_match' is not in camel case.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2651583495/issues/source?bid=3233864&fileBranchId=3324174#l480)